packages/framework/client-logger/fluid-telemetry/src/container/telemetryManager.ts

```diff
@@ -4,14 +4,15 @@
  */
 
 import { type ICriticalContainerError } from "@fluidframework/container-definitions";
-import { IContainerTelemetry, type ContainerHeartbeatTelemetry } from "./containerTelemetry.js";
+import { type IContainerTelemetry } from "./containerTelemetry.js";
 import { ContainerEventTelemetryProducer } from "./telemetryProducer.js";
-import { ITelemetryConsumer } from "../common/index.js";
+import { type ITelemetryConsumer } from "../common/index.js";
 import {
-	IFluidContainerSystemEventName,
+	type IFluidContainerSystemEventName,
 	IFluidContainerSystemEventNames,
 } from "./containerSystemEvents.js";
 import type { IFluidContainer } from "@fluidframework/fluid-static";
+import { ConnectionState } from "@fluidframework/container-loader";
```

Comment:
The type used for ConnectionState in IFluidContainer comes from container-definitions... it is identical to the one from container-loader today, but better if we import it from the same place that IFluidContainer does, so a mistake where those two get out of sync doesn't affect this.

Reply:
ConnectionState is a type not an enum from container-definitions so I can't use it the way I am, `this.container.connectionState === ConnectionState.Connected` instead I have to write `this.container.connectionState === 2` and add a comment for it to make sense. 

But I get your point, LMK if you think I should remove my dependency on container-loader and update the code.

Reply:
Oh, you're right... and I see that other places do import the enum from container-loader. We should be good here then, resolving.